onnxruntime/python/tools/quantization/base_quantizer.py

```diff
@@ -529,4 +529,6 @@ def adjust_tensor_ranges(self):
                 self.tensors_range[node.input[0]] = td
             # Adjust Softmax to range from 0.0 to 1.0
             elif node.op_type == "Softmax":
+                if not self.should_quantize_node(node):
+                    continue
```

Comment:
Could you add a test to reproduce the softmax bug?

Reply:
Thanks, added a unit test.